test/partiql-tests-runner/src/main/kotlin/org/partiql/runner/ReportAnalyzer.kt

```diff
@@ -118,6 +127,34 @@ class ReportAnalyzer(
         out.appendLine("- **IGNORED in $BASE but now PASSING in $TARGET**: ${ignoredFirstPassingSecond.count()}")
     }
 
+    private fun appendOptionalNewTests(out: Appendable, limit: Int) {
+        if (this.newPassing.isNotEmpty() || this.newFailing.isNotEmpty() || this.newIgnored.isNotEmpty()) {
+            out.appendMarkdown("## New Tests Added")
+            // character count limitation with comments in GitHub
+            // also, not ideal to list out hundreds of test names
+            if (newPassing.size + newFailing.size + newIgnored.size < limit) {
+                out.appendMarkdown("<details><summary>Click here to see</summary>")
+                var i: Int = 0
+                out.appendLine("### Passing $ICON_CHECK")
+                newPassing.forEach { testName ->
+                    out.appendLine("${i++}. $testName")
+                }
+                out.appendLine("### Failing $ICON_X")
+                newFailing.forEach { testName ->
+                    out.appendLine("${i++}. $testName")
+                }
+                out.appendLine("### Ignored $ICON_DIAMOND_ORANGE")
+                newIgnored.forEach { testName ->
+                    out.appendLine("${i++}. $testName")
```

Comment:
The sets are flipped here.
```suggestion
                newIgnored.forEach { testName ->
                    out.appendLine("${i++}. $testName")
                }
                out.appendLine("### Ignored $ICON_DIAMOND_ORANGE")
                newFailing.forEach { testName ->
                    out.appendLine("${i++}. $testName")
```

Reply:
hmm, It looks correct to me title comes before list.